### ISSUE TITLE
Increases S3 ListObjects 

### ISSUE THREAD

**Issue by shyamsunder00** [jeongukjae/tf-serving-s3#2]
While upgradaing from 2.1 to 2.11 with the patch, have observed s3 listobject frequency increased.Which thereby increased cost. 
Did we observed this behaviour? any way to control/remove it?
@jeongukjae

**Comment by jeongukjae** [comment]
@shyamsunder00 

I think this patch does not involve the frequency of s3 listobject operation.

https://github.com/tensorflow/serving/blob/master/tensorflow_serving/g3doc/serving_config.md#miscellaneous-flags

You can set `file_system_poll_wait_seconds` flag (the period with which the server polls the filesystem for new model versions) to adjust it.

**Comment by shyamsunder00** [comment]
Yes, have used the same config,but infact kept the same as TF2.1 .But observed increase in s3 listobject operation.
Not sure if its related to this patch or nature of TF2.11
@jeongukjae

**Comment by jeongukjae** [comment]
Oh, I got it.

You can check the patches in this file.
https://github.com/jeongukjae/tf-serving-s3/blob/main/tensorflow_serving.patch

And these lines are the patches to register the AWS S3 filesystem.
* https://github.com/jeongukjae/tf-serving-s3/blob/767c47e91ff352d921104fd32a7b71b458d53f36/tensorflow_serving.patch#L48-L71
* https://github.com/jeongukjae/tf-serving-s3/blob/767c47e91ff352d921104fd32a7b71b458d53f36/tensorflow_serving.patch#L107-L134

As you can see these lines, this patch does not contribute the behaviour of file system or something else. It just registers the S3 file system from tensorflow-io codebase. So maybe you have to dig in to tensorflow-io's s3 filesystem and s3 filesystem of older tensorflow versions to check this problem.
